src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/RemissionOption.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum RemissionOption {
+
+    ASYULM_SUPPORT_FROM_HOME_OFFICE("asylumSupportFromHo"),
```

Comment:
Typo here in ENUM >>  'ASYLUM'

Reply:
There's also a typo with the name of the handler , do you mind correcting it in this PR?

'appelantSubmittedWithRemissionRequestNotificationHandler' >> 'appellant'